Cognite.Simulator.Utils/Configuration.cs

```diff
@@ -84,6 +84,12 @@ public class ConnectorConfig
         /// </summary>
         public int StatusInterval { get; set; } = 10;
 
+        /// <summary>
+        /// The connector will check for license updates with this interval (in seconds)
+        /// </summary>
+        public int LicenseUpdateInterval { get; set; } = 30;
```

Comment:
Suggestion: By default, we seem to be checking for license for every 30 seconds, I feel this needs to be purely driven by config. If the config does not contain `license-update-interval`, then we should not be checking for license. Also we can keep minimum value as at least one hour?
@evertoncolling your thoughts?

Reply:
Changed LicenseUpdateInterval to 1 hour and added logic in GetLicenseCheckInterval() so that it would be minimum 1 hour. Also added EnableLicenseCheck in config so that check for license can be optional after it is implemented in each connector.

Reply:
I agree with the above. We should have a flag on the config file specifying whether to perform the check or not (`license-check` -> `enabled`), if this `enabled` flag is not present we default to not check the license. Additionally we should have an optional frequency parameter in the config file that specifies how frequent we check for a license (`license-check` -> `frequency`). If this `frequency` parameter is not present, we default to checking once per hour.

As a summary we should have 2 new config options (suggested names as below):
```yaml
license-check:
 enabled: true
 frequency: 60
```